src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Random random = new Random();
+        Colours[] colour = Colours.values();
+        int randomIndex = random.nextInt(colour.length - 1);
```

Comment:
The 'nextInt' method should not have '- 1' because it already generates a number from 0 (inclusive) to the specified value (exclusive), which is 'colour.length' in this case. With '- 1', you are excluding the last item of the enum from being selected.